sequencing-server/src/lib/codegen/CommandEDSLPreface.ts

```diff
@@ -854,19 +893,28 @@ export class Sequence implements SeqJson {
                 ...(r.metadata ? { metadata: r.metadata } : {}),
                 steps: [
                   r.steps[0].type === 'command'
-                    ? CommandStem.fromSeqJson(r.steps[0] as CommandStem,localNames, parameterNames)
-                    : r.steps[0].type === 'ground_block'
-                    ? // @ts-ignore : 'GroundBlock' found in JSON Spec
-                      Ground_Block.fromSeqJson(r.steps[0] as GroundBlock)
-                    : r.steps[0].type === 'ground_event'
-                    ? // @ts-ignore : 'GroundEvent' found in JSON Spec
-                      Ground_Event.fromSeqJson(r.steps[0] as GroundEvent)
-                    : r.steps[0],
+                      ? CommandStem.fromSeqJson(r.steps[0] as CommandStem,localNames, parameterNames)
+                      : r.steps[0].type === 'ground_block'
```

Comment:
I think it might be time to turn these types into an enum?

Reply:
I agree, I created a `enum`